libraries/scrcpy-decoder-webcodecs/src/video/decoder.ts

```
@@ -145,46 +160,59 @@ export class WebCodecsVideoDecoder implements ScrcpyVideoDecoder {
+                await this.#renderer.draw(frame);
+                frame.close();
+
+                this.#counter.increaseFramesDrawn();
+
+                if (this.#nextFrame) {
+                    frame = this.#nextFrame;
+                    this.#nextFrame = undefined;
+                } else {
+                    break;
+                }
+            } while (true);
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)